powersimdata/input/export_data.py

```diff
@@ -1,9 +1,13 @@
 import copy
+import warnings
 
 import numpy as np
+import pandas as pd
 from scipy.io import savemat
 
 from powersimdata.input.transform_profile import TransformProfile
+from powersimdata.input.grid import Grid
+from powersimdata.network.usa_tamu.constants import plants as plantconstants
```

Comment:
We don't want to assume that every grid will be a `usa_tamu` grid, since we're in the process of expanding our model coverage. To get the same set of information from an arbitrary grid, we can look at `Grid.model_immutables.plants`.